### ISSUE TITLE
Custom Word Tokenizer Error

### ISSUE THREAD

**Issue by bjarnigithub** [bitextor/bitextor#224]
Hi,

I'm using Bitextor version 8.1.0 in a Conda environment (the same error also occurs when using Bitextor in Docker (I think that's Bitextor version 8.1.1)) and I'm trying to use a custom word tokenizer with Bitextor, but Bitextor crashes on the following error:

```
Traceback (most recent call last):
  File "/home/user/anaconda3/envs/bitextor_conda_env/lib/python3.8/site-packages/bitextor/bitextor_tokenize.py", line 88, in <module>
    tokenized = tokenizer_func(content, tokenizer, lemmatizer).lower()
AttributeError: 'tuple' object has no attribute 'lower'
```

Here is the code I'm using, and I have also tried using the example custom word tokenizer code (the one that uses NLTK, i.e. https://github.com/bitextor/bitextor/blob/master/bitextor/example/nltk-word-tokeniser.py), but Bitextor crashes on the same error:

```
#!/usr/bin/env python3

from sys import stdin
from os import system as shell_command

with open("tokenization/sents_to_tokenize.txt", "w", encoding="utf-8") as sents_in:
        for line in stdin:
                sents_in.write(line)

sents_to_tokenize = open("tokenization/sents_to_tokenize.txt", "r", encoding="utf-8").read().split("\n")

for sent in sents_to_tokenize:
        shell_command("echo '{}' > tokenization/sent_to_tokenize.txt".format(sent.strip("\n")))
        shell_command("tokenize tokenization/sent_to_tokenize.txt tokenization/tokenized_sent.txt")

        tokenized_sentence = open("tokenization/tokenized_sent.txt", "r", encoding="utf-8").read()
        print(tokenized_sentence, end="")
```

This is the tokenizer I'm using in the code: https://github.com/mideind/Tokenizer. 

Why am I getting this error? How can I fix this so that I can use the custom word tokenizer code with Bitextor?

Thank you for your help 😊

**Comment by cgr71ii** [comment]
An easier solution should be to modify `/home/user/anaconda3/envs/bitextor_conda_env/lib/python3.8/site-packages/bitextor/bitextor_tokenize.py` and change

```python
tokenized = tokenizer_func(content, tokenizer, lemmatizer).lower()
```

to:

```python
tokenized = tokenizer_func(content, tokenizer, lemmatizer)[0].lower()
```

**Comment by cgr71ii** [comment]
Closing because this issue has been solved in version 8.2
